sdk/textanalytics/cognitiveservices-textanalytics/samples/recognizeEntities.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// NOTE: replace with import { TextAnalyticsClient } from "@azure/cognitiveservices-textanalytics"
+// in a standalone project
+import {
+  TextAnalyticsClient,
+  CognitiveServicesCredentials,
+  RecognizeEntitiesResult,
+  RecognizeEntitiesSuccessResult,
+  RecognizeEntitiesErrorResult
+} from "../src";
+
+export async function run() {
+  console.log(`Running recognizeEntities sample`);
+
+  // You will need to set these environment variables
+  const endPoint = process.env["AZ_CONFIG_ENDPOINT"]!;
+  const subscriptionKey = process.env["AZ_CONFIG_SUBSCRIPTION_KEY"]!;
+  const client = new TextAnalyticsClient(
+    endPoint,
+    new CognitiveServicesCredentials(subscriptionKey)
+  );
+
+  const [result] = await client.recognizeEntities(["I love living in Seattle."]);
+
+  if (isSuccess(result)) {
+    for (const entity of result.entities) {
+      console.log(`Found entity ${entity.text} of type ${entity.type}`);
+    }
+  }
+}
+
+function isSuccess(result: RecognizeEntitiesResult): result is RecognizeEntitiesSuccessResult {
+  return !(result as RecognizeEntitiesErrorResult).error;
+}
+
+// If you want to run this sample from a console
+// uncomment these lines so run() will get called
+run().catch((err) => {
```

Comment:
Do we want to comment these lines as per the comment above it ?

Reply:
There's ongoing discussion on the JS team about the best way to make the samples runnable individually as well as in CI, so yes I'll comment these for now but it will all change before this package ships its first preview. :)